packages/components/pie-webc-testing/src/playwright/fixtures.ts

```diff
@@ -0,0 +1,25 @@
+import { test as baseTest, expect as baseExpect } from '@sand4rt/experimental-ct-web';
+import { Page } from 'playwright-core';
+import AxeBuilder from '@axe-core/playwright';
+
+interface ExtendedTestContext {
+  page: Page;
+  makeAxeBuilder: () => AxeBuilder;
+}
+
+// Extend base test by providing "makeAxeBuilder"
+// This new "test" can be used in multiple test files, and each of them will get
+// a consistently configured AxeBuilder instance.
+export const test = baseTest.extend<ExtendedTestContext>({
+    makeAxeBuilder: [async ({ page }: { page: Page }, use: (builder: () => AxeBuilder) => Promise<void>) => {
+        const makeAxeBuilder = () => new AxeBuilder({ page })
+        .withTags(['wcag21a', 'wcag21aa', 'wcag143', 'cat.color', 'cat.aria'])
+        .disableRules(['color-contrast-enhanced']);
```

Comment:
Indentation seems a bit funny here. I don't know why ESLint didn't picked it but I'd already adjust it so it doesn't become a future issue.

Reply:
I'm afraid this might be an issue not related to this PR as the format on save doesn't suggest any issues so i assume it's aligned with out config.

Reply:
I'm afraid this might be an issue not related to this PR as the format on save doesn't suggest any issues so i assume it's aligned with out config.